### DISCUSSION TITLE
How to have several sessions?

### DISCUSSION THREAD

**Discussion by sawa-ko** [coder/code-server#4655] [Q&A]
For example, my friend connects to the web, but if I also connect to the web we both see the same editor, with the same open folder and files, I wonder how to make it so that when we both connect to each other, we can have different workspaces, and not have the same workspace both at the same time.

In short, I want to have my own editor instance in my browser and my friend to have his own editor instance in his browser, not both have the same editor instance all the time.

**Comment by lg-nineoneninethreenine** [comment]
Why not try Docker? That gives each of you a container so you can install Ubuntu on the container(there's an official image), install code-server and expose the container's port. But be wary of the costing!

**Reply by sawa-ko** [reply]
is that I want to have a single access domain, for example code.domain.com

**Reply by lg-nineoneninethreenine** [reply]
You could use the link feature in code-server, it makes a link only for you.

**Comment by jsjoeio** [comment]
I don't know exactly how you would do this. You might dig through the Discussions and see what you can find. I've heard some folks setting up one editor for multiple users like this for classroom settings.

**Reply by jsjoeio** [reply]
cc @bpmct another interesting use case
